CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # Changelog
 
+## [0.17.7](https://github.com/brave/browser-laptop/releases/tag/v0.17.7dev)
+- Added a new "Welcome" screen. ([#7821](https://github.com/brave/browser-laptop/issues/7821))
```

Comment:
I'd move this down since we aren't highlighting or showing it on first run in this release.